details/encryption.md

```diff
@@ -0,0 +1,396 @@
+# Interoperable Private Attribution Encryption
+
+Interoperable Private Attribution (IPA) relies on the encryption of data in
+order to meet its security and privacy goals.  This document describes how data
+is protected by encryption in IPA.
+
+
+# Communications Security
+
+IPA uses TLS ([RFC 8446](https://datatracker.ietf.org/doc/html/rfc8446)) for the
+protection of all network communications.  More concretely, this will involve
+the use of HTTPS ([RFC 9110](https://datatracker.ietf.org/doc/html/rfc9110)),
+with Web PKI authentication for servers.  Client authentication might be
+required in some cases, such as between helpers.
+
+Formally, this HTTPS only applies to those places where specifying
+interoperability requirements are specified, which is:
+
+* Between a record collector and the helper party network.
+
+* Between helpers in a helper party network.
+
+The expectation is that HTTPS or equivalent protection is used in some other
+interactions, as some privacy or security goals might not be achieved if data is
+not adequately protected, but this is out of scope for this document.  Other
+documents might specify expectations about how data is handled.
+
+
+# Match Key Protection
+
+Though TLS is a key component, IPA relies on an IND-CCA secure public key
+encryption scheme to protect match keys.  When a site requests an encrypted
+match key, the user agent generates three objects that are encrypted toward each
+of the three helpers ($P_1$, $P_2$, and $P_3$).
+
+HPKE ([RFC 9180](https://datatracker.ietf.org/doc/html/rfc9180)) is used to
+protect match keys.  HPKE provides an IND-CCA2 secure scheme that combines
+asymmetric and symmetric cryptography.
+
+
+## Helper Key Configuration
+
+Each helper party publishes an HPKE key configuration using the format described
```

Comment:
do we also need to formalize how UA obtain helpers public keys?

Reply:
Definitely.  This is an important component, but a difficult one.  This is a case where the system requires [correctness and consistency](https://datatracker.ietf.org/doc/html/draft-ietf-privacypass-key-consistency-00), otherwise the UA population might be segmented through the use of targeted key configurations.

For that reason, I'd left this out of scope for this piece.